electron-app/src/views/settings/General.vue

```diff
@@ -134,12 +145,21 @@ const { mapState, mapGetters } = createNamespacedHelpers('session');
   components: { MessageDialog },
   computed: {
     ...mapState(['settings']),
-    ...mapGetters(['currency'])
+    ...mapGetters(['currency']),
+    scrambleData: {
```

Comment:
Since we are using class components you could also do that directly in the class.

`get scrableData(): boolean {...}`
and
`set scrableData(value:boolean) {...}`

since this is how computed properties are done in the rest of the application it would be nice to keep it consistent.

Reply:
I guess in this case we don't even need the `mapState` mentioned above right? Only for `privacyMode`?

Reply:
If you don't want to `mapState`, because this would actually need a second computed property, this would be `this.$store.session.scrambleData` (note that TypeScript compiler might complain because we have incomplete typing here so you have to cast to any?/unknown to assess `session`).